opennms-config/src/test/resources/priority/eventconf2.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
```

Comment:
It doesn't look like anything references this file.

Reply:
added missing test that uses this